utils/package_installer

```diff
@@ -173,7 +151,147 @@ check_and_install_kubeblocks() {
     if ! kbcli cluster list 2>/dev/null || ! kbcli kubeblocks status; then
         gum spin --title "installing kubeblocks" --show-output -- kbcli kubeblocks install --set image.registry="docker.io"
     fi
+   
+    extract_kubeblocks_cli_version
     
-    display_package_version "kubeblocks" "kbcli version | grep KubeBlocks"
+}
+
+
+
+extract_grapple_cli_version(){
```

Comment:
there is one caveat in this.
Since we can't save the version of grpl in the VERSION file while the grapple package is being installed or upgraded, 
we are doing a curl to fetch the latest release info from GitHub and then we extract the latest release tag from it.
The problem with this is that when we first run 
grpl v
this function ran, it checks that grpl version doesn't exist in the file, runs the curl to get the latest version
we store that in the VERSION file and when grpl v is run again we don't do crul again, we show the version from the file. 

So the problem is what will happen when grpl is upgraded?
The file will still have grpl version in the VERSION file so check
  if ! grep -q "grpl" "$GRPL_WORKDIR/VERSION" >/dev/null 2>&1; then 
won't become true and the old version will be displayed.

So what if we always fetch the latest version from GitHub instead of storing it in the file? It seems this will resolve the issue, right? But no, because now we run into another issue. There is a limited number of API calls we can do an hour so if the version command runs more than that then we'll get the following error

{"message":"API rate limit exceeded for 154.192.74.14. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)","documentation_url":"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}

So what do you suggest?
Here is what I think is the correct solution

- Our package shouldn't always install the latest release
- We should decide which version should be installed by statically adding in to the homebrew formula (that's how civo cli do this) and also updating the version file manually

Reply:
But if we were to upgrade version using grpl version as mentioned in this ticket, and write the new version into the file
https://wolke7.atlassian.net/browse/GC-23
then the version problem will be resolved

Reply:
@zahid-iqbal-alpha 
I would just include the version in the deployment of grpl (in the git repo as a version file).
for the time being, we can update it manually in the code.
with time, I would include a CI job, that automatically updates the version inside the code accordingly...

does that make sense?